test/pushstate-iframe-tests-modern.js

```diff
@@ -0,0 +1,7 @@
+import { makeTest } from "./pushstate-iframe-tests";
+
+if (window.history && history.pushState) {
+	makeTest("can-map");
+	makeTest("can-define/map/map");
+	makeTest("can-observable-object");
+}
```

Comment:
I wasn’t sure what to call these files… suggestions welcome. I went with this:

- `pushstate-iframe-tests-ie`: for IE to run the iframe tests
- `pushstate-iframe-tests-modern`: for everything else to run the iframe tests
- `pushstate-iframe-tests`: has all the actual iframe tests and exports the helper the other files use
- `pushstate-tests-ie`: for IE to run the rest of the tests
- `pushstate-tests-modern`: for everything else to run the rest of the tests
- `pushstate-tests`: has the rest of the actual tests and exports the helper the other files use